README

```diff
@@ -732,7 +730,7 @@ Disable with 0
 Default value is 10
 
 XLIO_SELECT_SKIP_OS
-Similar to XLIO_RX_SKIP_OS, but in select() or poll() this will force the XLIO
+For select() or poll() this will force XLIO
 to check the non offloaded fd even though an offloaded socket has ready
 packets found while polling.
```

Comment:
Merge the lines and make them up to 80 symbols in a row.

Reply:
Done